eslint/babel-eslint-tests/test/integration/config-files.js

```diff
@@ -1,35 +1,41 @@
 import { ESLint } from "eslint";
-import path from "path";
 import { fileURLToPath } from "url";
-import { itESM, itGteNoESM } from "$repo-utils";
 
 describe("Babel config files", () => {
-  const nodeGte12NoESM = itGteNoESM("12.0.0");
-
-  itESM("works with babel.config.mjs", async () => {
-    const engine = new ESLint({ ignore: false });
-    expect(
-      await engine.lintFiles([
-        path.resolve(
-          path.dirname(fileURLToPath(import.meta.url)),
-          `../fixtures/mjs-config-file/a.js`,
+  it("works with babel.config.mjs - ESLint " + ESLint.version, async () => {
+    if (parseInt(ESLint.version, 10) >= 9) {
+      const engine = new ESLint({
+        ignore: false,
+        overrideConfigFile: fileURLToPath(
+          new URL(
+            "../fixtures/mjs-config-file/eslint.config.js",
```

Comment:
The fixture `fixtures/mjs-config-file` is identical to `fixtures/mjs-config-file-babel-7`, here we reuse the fixture and merge these two tests as ESLint tests will be run against Node >= 12 only.